commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { extends: ['@commitlint/config-angular'] };
```

Comment:
This commit linting is necessary to trigger automatically semantically-versioned builds via `semantic-release`.